app/assets/images/govuk_publishing_components/action-link-arrow--transparent.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="39" height="39" viewBox="0 0 39 39" fill="none" xmlns="http://www.w3.org/2000/svg">
+  <path style="fill:#fff500;" d="M 17.009766,6.8183592 14.181641,9.6464838 22.035156,17.5 0.03515625,17.566406 0,21.5 h 22.035156 l -7.853515,7.853516 2.828125,2.828125 L 29.691406,19.5 Z M 39,19.5 C 39,30.269553 30.269553,39 19.5,39 9.3731566,39 1.0492998,31.280497 0.09185841,21.404918 0.03109354,20.778156 0,20.142709 0,19.5 0,8.7304474 8.7304474,0 19.5,0 30.269553,0 39,8.7304474 39,19.5 Z"/>
```

Comment:
Can this be compressed any further? Seems to be a couple of newlines and a tab that aren't needed? Or do we care about readability here?

Reply:
Running it through https://jakearchibald.github.io/svgomg/ takes it down by 33%:

``` svg
<svg width="39" height="39" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M17.01 6.818l-2.828 2.828 7.853 7.854-22 .066L0 21.5h22.035l-7.853 7.854 2.828 2.828L29.69 19.5zM39 19.5C39 30.27 30.27 39 19.5 39 9.373 39 1.05 31.28.092 21.405A19.737 19.737 0 010 19.5C0 8.73 8.73 0 19.5 0S39 8.73 39 19.5z" fill="#fff500"/></svg>
```

Reply:
This is the compressed version (in terms of polygons) – couldn't make it smaller than this, but open to suggestions. I'll remove the white space, although I'm not sure it makes a difference overall.

Reply:
Compressed further; thanks both! 🗜